### PR TITLE
[Homework #19] React-api by Abraham Trenyan

### PR DESCRIPTION
**Pull request by AbrahamTrenyan**
Hi Vitali! I've been struggling mostly with the percentage functionality, but finally managed to make it work.

### CODE REVIEW

**Comment by VitalikMelnyk** [comment]
It necessary some fixes:
1. Format all files with Prettier.
2. Fix the design. More info on the screenshot. It is about one text is overlapped on another, and when a lot of numbers, we have an overflow and don't see all numbers and operations.
Screenshot 2022-09-30 at 12 24 23

File: homeworks/abraham-trenyan/19-react-api/src/App.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import { React, useState } from 'react';
+import './App.css';
+import Container from './Components/Container/Container';
+import Screen from './Components/Screen/Screen';
+import Keyboard from './Components/Keyboard/Keyboard';
+import Button from './Components/Button/Button';
+function App() {
+  const [buttons] = useState([
+    'C',
```
Comment (VitalikMelnyk) [inline homeworks/abraham-trenyan/19-react-api/src/App.js:9]: Export to constants in `utils` or `helpers` folder

File: homeworks/abraham-trenyan/19-react-api/src/App.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      percentOperation = Number((numberToDivideByHundred) / 100);  // Will store the result of the percentage.
+      setResult(percentOperation.toString().split(''));  // Generates an array with the result number stringified, so we can do further operations upon this result/operation.
+      setOperation(percentOperation.toString().split(''));
+    } else {
+      // If the percentage sign is found after another sign, it would behave like this: takes the number before the percentage sign and divides it by 100, then multiplies it by the number before the other sign, because it will be a percentage of this other number.
+      let numberToDivideByHundred = percentOperation.slice(lastSignIndex + 1, percentIndex).join(''); // Takes the part of the operation since the last sign to the percentage sign, that is the number affected by the sign that we should divide by 100.
+      let multiplier = Function('return ' + percentOperation.slice(0, lastSignIndex).join(''))();
```
Comment (VitalikMelnyk) [inline homeworks/abraham-trenyan/19-react-api/src/App.js:73]: Screenshot 2022-09-30 at 12 34 57
In the future should be cautious with this.

File: homeworks/abraham-trenyan/19-react-api/src/App.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  return (
+    <Container>
+      <Screen input={operation} output={result} />
+      <Keyboard>
+        {buttons.map((btn, i) => (
+          <Button
+            key={i}
+            value={btn}
+            onClick={
+              btn === 'C'
```
Comment (VitalikMelnyk) [inline homeworks/abraham-trenyan/19-react-api/src/App.js:101]: Refactor this because it isn't readable, as for me.

**Comment by AbrahamTrenyan** [comment]
I set a limit for numbers to avoid overflow, fixed mobile design, format files with prettier, moved buttons to utils folder, renamed components folder  and refactored onclick function. Thanks for the comments!

**Review by VitalikMelnyk — APPROVED** [review]

**Pull request merged**